[CN-698] Update Helm installation commands with restricted and cluster wide installatio modes (#74)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/get-started.adoc

+
NOTE: Since CRDs are global resources, they may need to be installed by an administrator.

+
[tabs]
====
Cluster Wide Installation::
+
--

> Run the following command to *deploy the operator and the CRDs together*. Operator by default watches all namespaces, you can configure the watched namespace with `watchNamespace` variable.

Reviewer: ```suggestion
Run the following command to *deploy the Operator and the CRDs together*. By default, the Hazelcast Platform Operator watches all namespaces. Use the `watchNamespace` variable to update this configuration.
```

[source,shell]
----
helm install operator hazelcast/hazelcast-platform-operator \
    --set=installCRDs=true \
    --set=watchNamespace="*"
----
--

Restricted Installation::
+
--

> Run the following commands to *deploy the operator and the CRDs separately*. Following operation may be done by an administrator.

Reviewer: ```suggestion
Run the following commands to *deploy the Operator and the CRDs separately*. An administrator may need to do this.
```

[source,shell]
----
helm install operator-crds hazelcast/hazelcast-platform-operator-crds
----

> After installing CRDs, install the operator by running the following command. Following operation requires only namespace scoped permissions for `hz-system` and `watched-namespace` namespaces.

Reviewer: ```suggestion
After installing CRDs, install the Operator by running the following command. This operation requires only namespace-scoped permissions for `hz-system` and `watched-namespace` namespaces.
```
